packages/features/ee/payments/api/webhook.ts

```diff
@@ -335,7 +337,14 @@ const handleSetupSuccess = async (event: Stripe.Event) => {
   // If the card information was already captured in the same customer. Delete the previous payment method
 
   if (!eventTypeRaw?.requiresConfirmation) {
```

Comment:
Does it work correctly when Requires Confirmation is conditional? i.e. when it returns true but it actually doesn't require confirmation.


I see this logic in handleNewBooking which isn't here.
<img width="1064" alt="Screenshot 2023-04-25 at 1 43 12 PM" src="https://user-images.githubusercontent.com/1780212/234215880-fd9f8de4-400a-47f9-a154-692ea13a45fb.png">